src/app/common/utils/open-in-new-tab.ts

```diff
@@ -11,7 +11,7 @@ export function openInNewTab(url: string) {
 
 export function openIndexPageInNewTab(path: RouteUrls | string, searchParams?: string) {
   const paramsString = searchParams ? searchParams : '';
-  return chrome.tabs.create({
```

Comment:
why this removal?

Reply:
I did this change early on bc I get this now with the return of `Tab` rather than `void` from the fn, but maybe that isn't the right way to fix it? I can look at it again now that things seem to be working a bit better overall...

<img width="1067" alt="Screenshot 2025-06-23 at 3 03 20 PM" src="https://github.com/user-attachments/assets/5aefd6e8-a921-4d44-a8a4-2b8d872b7f46" />

Reply:
If I change to this the type error goes away...

```ts
return whenWallet({
      ledger: () =>
        whenPageMode({
          full: () => void navigate(routeUrl),
          popup: () => void openIndexPageInNewTab(routeUrl),
        })(),
      software: () => navigate(routeUrl),
    })();
```